### PR TITLE
exio-2146-display utc time to local time

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EXUI-2146
https://tools.hmcts.net/jira/browse/EXUI-2230

### Change description ###
Case File View date/time as GMT rather than Local BST

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by anthonydummer — APPROVED** [review]
LGTM

**Review by olusegz07 — APPROVED** [review]

**Review by Josh-HMCTS — APPROVED** [review]

**Review by andywilkinshmcts — APPROVED** [review]

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]
I'm not sure if this will do the correct local time to UTC conversion. Are there any tests for this? You need tests with times, checking that the displayed value is offset during BST

**Review by OgunyemiO — DISMISSED** [review]

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts [hunk 1]
```diff
@@ -34,26 +36,50 @@ describe('CaseFileViewFolderComponent', () => {
       document_filename: 'Lager encyclopedia',
       document_binary_url: '/test/binary',
       attribute_path: '',
-      upload_timestamp: '11 May 2023 00:00:00'
+      upload_timestamp: '2023-05-11T11:15:10.00'
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:39]: Have you checked that this is the format returned by the DocumentsAndCategories API?

**Review by andywilkinshmcts — COMMENTED** [review]
Fine assuming that the ISO format that you are now using is that returned by CCD

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts (hunk 1 above)
Comment (RiteshHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:39]: Yes @andywilkinshmcts

**Review by andywilkinshmcts — APPROVED** [review]

**Pull request merged**